.gitignore

```diff
@@ -3,6 +3,8 @@ perf/
 work/
 download/
 build/
+ninja-build/
+faasm.ini
```

Comment:
`faasm.ini` shouldn't be in the root of the project, it should live at `~/faasm/faasm.ini`. I don't mind leaving it in `gitignore` but worth checking if you really need it there

Reply:
I happened to check out faasm at `~/` on my VM!